Controller/QueueController.php

```diff
@@ -43,11 +44,11 @@ public function admin_reset() {
 		$this->request->allowMethod('post');
 		$res = $this->QueuedTask->truncate();
 		if ($res) {
-			$this->Flash->message(__d('queue', 'OK'), 'success');
```

Comment:
The plugin uses the Tools functionality. Is it possible to check if the component(s) exist and conditionally use the default syntax only if not?